### PR TITLE
PPR.1 and 2 submission for review

### PR DESCRIPTION
**Pull request by jni2000**
Please review PPR.1 and PPR.2

### CODE REVIEW

File: src/harness/testcases/WINNF_FT_S_PPR_testcase.py [hunk 1]
```diff
@@ -0,0 +1,391 @@
...
+    }
+    cbsd_records_domain_proxy_1 = {
+        'registrationRequests': [device_3],
+        'grantRequests': [grant_request_3]
+    }
+
+    # Protected entity record
+    protected_entities = {
+        'palRecords': pal_records_1,
+        'ppaRecords': [ppa_record_1]
```
Comment (harinineelu) [inline src/harness/testcases/WINNF_FT_S_PPR_testcase.py]: The spec says 'A PPA is injected in the SAS before...'.

Need to be just ppa_record_1 (not a list) and the field name would also be singular 'ppaRecord'. 

(Same in PPR.2)

File: src/harness/testcases/WINNF_FT_S_PPR_testcase.py [hunk 2]
```diff
@@ -0,0 +1,391 @@
...
+    del device_4['measCapability']
+
+    # Load PPA record
+    ppa_record = json.load(
+      open(os.path.join('testcases', 'testdata', 'ppa_record_0.json')))
+    pal_record = json.load(
+      open(os.path.join('testcases', 'testdata', 'pal_record_0.json')))
+
+    pal_low_frequency = 3550000000
+    pal_high_frequency = 3570000000
```
Comment (harinineelu) [inline src/harness/testcases/WINNF_FT_S_PPR_testcase.py]: pal_high_frequency = 3570000000. 

- Needs to be 3560000000 ? 
(Same in PPR.2)

File: src/harness/testcases/WINNF_FT_S_PPR_testcase.py [hunk 3]
```diff
@@ -0,0 +1,391 @@
...
+
+    device_2 = json.load(
+      open(os.path.join('testcases', 'testdata', 'device_b.json')))
+    # Moving device_2 to a location outside 40 meters of PPA zone
+    device_2['installationParam']['latitude'] = 38.8201
+    device_2['installationParam']['longitude'] = -97.27651
+
+    device_3 = json.load(
+      open(os.path.join('testcases', 'testdata', 'device_c.json')))
+    # Moving device_3 to a location within PPA zone
```
Comment (harinineelu) [inline src/harness/testcases/WINNF_FT_S_PPR_testcase.py]: Using getRandomLatLongInPolygon(ppa_record_1) might be a better choice? (Same in PPR.2)

File: src/harness/testcases/WINNF_FT_S_PPR_testcase.py [hunk 4]
```diff
@@ -0,0 +1,391 @@
...
+    # Load devices info
+    device_1 = json.load(
+      open(os.path.join('testcases', 'testdata', 'device_a.json')))
+    # Moving device_1 to a location within 40 meters of PPA zone
+    device_1['installationParam']['latitude'] = 38.8203
+    device_1['installationParam']['longitude'] = -97.2741
+
+    device_2 = json.load(
+      open(os.path.join('testcases', 'testdata', 'device_b.json')))
+    # Moving device_2 to a location outside 40 meters of PPA zone
```
Comment (harinineelu) [inline src/harness/testcases/WINNF_FT_S_PPR_testcase.py]: meters -> km. (Same in PPR.2)

File: src/harness/testcases/WINNF_FT_S_PPR_testcase.py (hunk 1 above)
Comment (jni2000) [reply src/harness/testcases/WINNF_FT_S_PPR_testcase.py]: Modified   ppaRecord to be singular

File: src/harness/testcases/WINNF_FT_S_PPR_testcase.py (hunk 2 above)
Comment (jni2000) [reply src/harness/testcases/WINNF_FT_S_PPR_testcase.py]: Modified   pal_high_frequency to 3570000000 for both testcases

File: src/harness/testcases/WINNF_FT_S_PPR_testcase.py (hunk 3 above)
Comment (jni2000) [reply src/harness/testcases/WINNF_FT_S_PPR_testcase.py]: As   suggested, modified to use “getRandomLatLongInPolygon” for both testcases

File: src/harness/testcases/WINNF_FT_S_PPR_testcase.py (hunk 4 above)
Comment (jni2000) [reply src/harness/testcases/WINNF_FT_S_PPR_testcase.py]: Modified meters to KMs for both testcases

**Comment by jni2000** [comment]
harinineelu

Thanks for reviewing and comments. We have addressed your latest comments and made required changes and fixes. Please continue.

**Pull request merged**